sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_operationid.json

```diff
@@ -273,7 +273,7 @@
             "post": {
                 "summary": "Create, update, or delete the identities of multiple devices from the IoT hub identity registry.",
                 "description": "Create, update, or delete the identiies of multiple devices from the IoT hub identity registry. A device identity can be specified only once in the list. Different operations (create, update, delete) on different devices are allowed. A maximum of 100 devices can be specified per invocation. For large scale operations, consider using the import feature using blob storage(https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities). For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
-                "operationId": "RegistryManager_BulkDeviceCRUD",
+                "operationId": "Devices_BulkDeviceCRUD",
```

Comment:
We'd never want to expose a method called BulkDevice**CRUD**. What verb can we use instead?

Reply:
BulkDeviceOpeations? I am open to any suggestions.